test/functional/test_site/testDates.md

```diff
@@ -0,0 +1,27 @@
+## Dates
```

Comment:
Could we move some of these to `variables.md` to ensure they work in `site.js`?

Reply:
Could move some of the filters to `variables.md` as well, to ensure the filter works in it

```
{{ base2 | date }} should be Tue 1 Jan

{{ base2 | date(format1) }} should be 01 01 2019
```
maybe